evm/src/cpu/kernel/asm/secp256k1/curve_add.asm

```diff
@@ -0,0 +1,338 @@
+// #define N 0xfffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8cd0364141 // Secp256k1 scalar field order
+
+// Secp256k1 elliptic curve addition.
+// Uses the standard affine addition formula.
+global ec_add_secp:
```

Comment:
Hm not sure if we'll use this, since there isn't a precompile for secp add? But we can keep it for now if you like

Reply:
Do you mean we delete this one and keep only `ec_add_valid_points_secp`?

Reply:
Yeah that's what I mean

Reply:
Ok I deleted it and also `ec_mul_secp` since it's also unused.

Reply:
Ok I deleted it and also `ec_mul_secp` since it's also unused.